CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## master (unreleased)
 
+### New features
+
+* [#6759](https://github.com/rubocop-hq/rubocop/issues/6759): Mention configuration in Basic Usage documentation. ([@scottmatthewman][])
```

Comment:
Typically we don't mention documentation changes in the changelog.

Reply:
Cool. I'll reverse that out, and add the link to rubystyle.guide as well.

For review purposes, how do you prefer to deal with amendments? Amending the original commits, or submitting new commits and squashing once accepted by the review?

I know it's pretty academic in this small case, but I hope to be contributing a little bit more in future :)

Reply:
> submitting new commits and squashing once accepted by the review

That's my preference.

> I know it's pretty academic in this small case, but I hope to be contributing a little bit more in future 😄

Glad to hear this! We're always looking for more contributors! :-)